core/storage/suggestion/gae_models.py

```diff
@@ -1292,3 +1292,678 @@ def export_data(
                     date.isoformat() for date in model.contribution_dates]
             }
         return user_data
+
+
+class TranslationReviewStatsModel(base_models.BaseModel):
+    """Records the contributor dashboard translation review stats. There
+    is one instance of this model per (language_code, reviewer_user_id,
+    topic_id) tuple. See related design doc for more details:
+    """
+
+    # We use the model id as a key in the Takeout dict.
+    ID_IS_USED_AS_TAKEOUT_KEY = True
+
+    # The ISO 639-1 language code for which the translation reviews were
+    # made.
+    language_code = datastore_services.StringProperty(
+        required=True, indexed=True)
+    # The user ID of the translation reviewer.
+    contributor_user_id = datastore_services.StringProperty(
+        required=True, indexed=True)
+    # The topic ID of the translation reviews.
+    topic_id = datastore_services.StringProperty(required=True, indexed=True)
+    # The number of reviewed translations.
+    reviewed_translations_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The total word count of reviewed translations. Excludes HTML tags and
+    # attributes.
+    reviewed_translation_word_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The number of accepted translations.
+    accepted_translations_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The number of accepted translations with reviewer edits.
+    accepted_translations_with_reviewer_edits_count = (
+        datastore_services.IntegerProperty(required=True, indexed=True))
+    # The total word count of accepted translations. Excludes HTML tags and
+    # attributes.
+    accepted_translation_word_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The total word count of accepted translations with reviewer edits.
+    # Excludes HTML tags and attributes.
+    accepted_translations_with_reviewer_edits_word_count = (
+        datastore_services.IntegerProperty(required=True, indexed=True))
+    # The unique last_updated dates of the translation reviews.
+    review_months = datastore_services.StringProperty(
+        repeated=True, indexed=True)
+    # The first date that the reviewer made a translation review.
+    first_contribution_date = datastore_services.DateProperty(indexed=True)
+    # The last date that the reviewer made a translation review.
+    last_contribution_date = datastore_services.DateProperty(indexed=True)
```

Comment:
Why are these necessary again? I don't see the dates surfaced in the mocks anywhere?

Reply:
As we discussed in the meeting, the first contribution date and the last contribution date will help to get the range(such as 2022 Apr - 2022 Jun) and display them in the front end. Also dropped the contribution/review month fields since the range of months can be taken from these two fields. https://www.figma.com/file/SNOHAatFcCvdvIAdxAzF6E/Contributor-Dashboard-Achievements?node-id=107%3A2